javascript/mcp-server/package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "forevervm-mcp-server",
-  "version": "0.0.0",
-  "private": true,
+  "name": "forevervm-mcp",
+  "version": "0.1.20",
   "description": "",
   "main": "index.js",
   "type": "module",
   "bin": {
-    "forevervm": "./build/index.js"
+    "forevervm-mcp": "./build/index.js"
```

Comment:
I think for `npx` this needs to match the package name?

Reply:
I _think_ that's right